gems/pending/spec/openstack/amqp/openstack_rabbit_event_monitor_spec.rb

```diff
@@ -48,4 +46,25 @@
     # get back to this piece
   end
 
+  context "remove legacy queues" do
+    before :each do
+    end
+
+    it "successfully removes legacy queues" do
```

Comment:
I personally would just remove this and make the outer context the `it`.  The extra description doesn't say anything over and above what the method name already tells us.